config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -14,6 +14,8 @@ presubmits:
     labels:
       preset-service-account: "true"
       preset-dind-enabled: "true"
+    decoration_config:
+      timeout: 3h
```

Comment:
3 hours sounds a lot to me. Isn't 2 hours really enough?

Reply:
I agree with jordan in https://github.com/kubernetes/kubernetes/pull/90639#issuecomment-621810350 we should consider splitting the verify job up into e.g. verify and verify-slow